packages/g/gimp-help/package.yml

```diff
@@ -1,72 +1,163 @@
+          - /usr/share/gimp/*/help/pdf/gimp-keys-da.pdf
+    - de : 
+          - /usr/share/gimp/*/help/de
+          - /usr/share/gimp/*/help/pdf/gimp-keys-de.pdf
+    - el : 
+          - /usr/share/gimp/*/help/el
+          - /usr/share/gimp/*/help/pdf/gimp-keys-el.pdf
+    - en_gb : 
+          - /usr/share/gimp/*/help/en_GB
+          - /usr/share/gimp/*/help/pdf/gimp-keys-en_GB.pdf
+    - es :
+          - /usr/share/gimp/*/help/es
+          - /usr/share/gimp/*/help/pdf/gimp-keys-es.pdf
+    - fa :
+          - /usr/share/gimp/*/help/fa
+          - /usr/share/gimp/*/help/pdf/gimp-keys-fa.pdf
+    - fi : 
+          - /usr/share/gimp/*/help/fi
+          - /usr/share/gimp/*/help/pdf/gimp-keys-fi.pdf
+    - fr : 
+          - /usr/share/gimp/*/help/fr
+          - /usr/share/gimp/*/help/pdf/gimp-keys-fr.pdf
+    - hr :
+          - /usr/share/gimp/*/help/hr
+          - /usr/share/gimp/*/help/pdf/gimp-keys-hr.pdf
+    - hu : 
+          - /usr/share/gimp/*/help/hu
+          - /usr/share/gimp/*/help/pdf/gimp-keys-hu.pdf
+    - it :
+          - /usr/share/gimp/*/help/it
+          - /usr/share/gimp/*/help/pdf/gimp-keys-it.pdf
+    - ja : 
+          - /usr/share/gimp/*/help/ja
+          - /usr/share/gimp/*/help/pdf/gimp-keys-ja.pdf
+    - ko : 
+          - /usr/share/gimp/*/help/ko
+          - /usr/share/gimp/*/help/pdf/gimp-keys-ko.pdf
+    - lt : 
+          - /usr/share/gimp/*/help/lt
+          - /usr/share/gimp/*/help/pdf/gimp-keys-lt.pdf
+    - nl : 
+          - /usr/share/gimp/*/help/nl
+          - /usr/share/gimp/*/help/pdf/gimp-keys-nl.pdf
+    - nn : 
+          - /usr/share/gimp/*/help/nn
+          - /usr/share/gimp/*/help/pdf/gimp-keys-nn.pdf
+    #- pl : in 2.10.34 there is only the gimp-keys-pl
+    #      - /usr/share/gimp/*/help/pl
+    #      - /usr/share/gimp/*/help/pdf/gimp-keys-pl.pdf    
+    - pt :
+          - /usr/share/gimp/*/help/pt
+          - /usr/share/gimp/*/help/pdf/gimp-keys-pt.pdf
+    - pt_br :
+          - /usr/share/gimp/*/help/pt_BR
+          - /usr/share/gimp/*/help/pdf/gimp-keys-pt_BR.pdf
+    - ro : 
+          - /usr/share/gimp/*/help/ro
+          - /usr/share/gimp/*/help/pdf/gimp-keys-ro.pdf
+    - ru :
+          - /usr/share/gimp/*/help/ru
+          - /usr/share/gimp/*/help/pdf/gimp-keys-ru.pdf
+    - sl :
+          - /usr/share/gimp/*/help/sl
+          - /usr/share/gimp/*/help/pdf/gimp-keys-sl.pdf
+    - sv :
+          - /usr/share/gimp/*/help/sv
+          - /usr/share/gimp/*/help/pdf/gimp-keys-sv.pdf
+    - uk :
+          - /usr/share/gimp/*/help/uk
+          - /usr/share/gimp/*/help/pdf/gimp-keys-uk.pdf
+    - zh_cn :
+          - /usr/share/gimp/*/help/zh_CN
+          - /usr/share/gimp/*/help/pdf/gimp-keys-zh_CN.pdf
 builddeps  :
     - pkgconfig(gimp-2.0)
     - docbook-xml
+    #The following have been added to populate the Quickreferences
+    - pkgconfig(librsvg-2.0)
+    - font-noto-cjk
 setup      : |
     %configure
 build      : |
```

Comment:
I built this locally on my system that has 16 cores (32 threads) and observed that the system ran out of memory and OOMkilled the build. I lowered it to 16GB and observed a memory usage of ~80GB, higher than the build server has. Please replace the `build` section with the following:
```
    # The memory demands for this build are very high. -j16 was observed consuming ~80GB of memory, more than the build server has.
    # Limit the jobs to 8 so as to not overwhelm the build server.
    MAX_JOBS=%YJOBS%
    if [ $MAX_JOBS -gt 8 ]; then
        MAX_JOBS=8
    fi
    make -j$MAX_JOBS
```

Reply:
Yes, in fact I had to change eopkg.conf from jobs=auto to jobs=3...

Reply:
Done